ui/lib/kv/addon/templates/secret/metadata/index.hbs

```diff
@@ -4,8 +4,14 @@
 ~}}
 
 <Page::Secret::Metadata::Details
+  @backend={{this.model.backend}}
+  @breadcrumbs={{this.breadcrumbs}}
+  @canDeleteMetadata={{this.model.permissions.metadata.canDelete}}
+  @canReadCustomMetadata={{or this.model.permissions.metadata.canRead this.model.permissions.data.canRead}}
```

Comment:
That sneaky API and overlap. The `or` at first here didn't make sense to me because if the attr is `canReadCustomMetadata` then I would think we should only check read on metadata. Then I remembered that there's overlap that you set something on metadata and can read it on data. Maybe a comment here or in the component doc mentioning the overlap?